src/main/java/com/rackspacecloud/metrics/ingestionservice/listeners/rawlisteners/RawListener.java

```diff
@@ -135,7 +136,7 @@ private void writeIntoInfluxDb(Map<TenantIdAndMeasurement, List<String>> tenantP
                         tenantIdAndMeasurement.getMeasurement(), "full");
                 // TODO: make enum for rollup level
 
-            } catch (Exception e) {
+            } catch (IOException e) {
                 String msg = String.format("Write to InfluxDB failed with exception message [%s].", e.getMessage());
                 log.error("[{}] Payload [{}] \n [{}]", msg, payload, e);
```

Comment:
While you're in here, this mixed use of `String.format` and slf4j placeholders is silly. It should only be using slf4j placeholders and not pre-include the message of the exception since the `, e);` will enable that visibility.